iStore/ContentView.swift

```diff
@@ -15,19 +15,14 @@ struct ContentView: View {
                     Image(systemName: "house.fill")
                 }
             
-            CategoriesView()
```

Comment:
I removed CategoriesView here because I will remove the CategoriesView entirely from the app, as the backend doesn't properly support the intended purpose of that View.

I will remove all the related Views in another Pull Request.